### DISCUSSION TITLE
flash.nvim maps R to Treesitter Search. How to enter Replace mode?

### DISCUSSION THREAD

**Discussion by ddbrierton** [LazyVim/LazyVim#994] [Q&A] [answered]
I’ve been using and enjoying the new flash.nvim package. It maps `R` to something, which is fine, I very rarely use replace mode, but today I wanted to quickly use it, and I couldn’t figure out how else you can change into replace mode. I read through what I thought were the relevant help pages, and they only mention using `R`, and the same searching on Google. Is there a command I can type?

Thanks in advance.

**Comment by folke** [comment, marked as answer]
I just pushed an update to not set `R` in normal mode. Maybe some people do actually use that often.

To select with treesitter search, you should now do `vR` instead of just `R`.

In case you still want to know about your orig question, you could do somethning like `:normal! R`
